scratch.go

```diff
@@ -417,13 +450,18 @@ func createLayout() error {
 	})
 }
 
-func prepare(config *Config, docker string, args ...string) error {
+func firstPrepare() error {
 	os.Setenv("PATH", "/sbin:/usr/sbin:/usr/bin")
 
 	if err := defaultFiles(
 		"/etc/ssl/certs/ca-certificates.crt",
 		"/etc/passwd",
 		"/etc/group",
+		"/etc/selinux/config",
+		"/etc/selinux/ros/policy/policy.29",
```

Comment:
Maybe we should just copy the directory and not refer the to .29 files.  If it's too much of a pain, no big deal.